gsoc/GSOC_2017_Application_proposal.md

```diff
@@ -0,0 +1,70 @@
+## Google Summer of Code - GSOC_2017_Application_proposal
+
+### Student
+
+**Name:** Guohui Ding
+
+**Affiliation:** University of Colorado Boulder, Computer Science, PhD
+
+**Location:** Boulder CO, USA, MST
+
+**GitHub handle:** @ikol1729
+
+**Slack handle:** @guohui
+
+**Email:** guohui.ding@colorado.edu
+
+**Other Contact:**  https://www.linkedin.com/in/guohuiding
+
+### Project Details
+
+**Project Title:**  Apply Machine Learning to Monitoring Website Changes
+
+**Project Abstract/Summary (<20 words):**  Improve the website monitoring system by using machine learning to identify the significant changes of the websites.
+
+**Describe the need your project fills:**  
+The broader goal of the project is to build an assistant system to reduce the work of analysts. This kind of reduction means: 1) Deal with the data computation with preprocessing and reduce all unnecessary extra work so that the system can provide the useful information as much as possible to the analysts. 2) Detect the changes of the target websites and identify the significance of these website changes accurately and efficiently.
+
+**Describe how your project meets this need:**  
+To reach the goal above, the project overview has already shown a very clear picture of the use case and the corresponding definitions, architecture etc. But there are still some places that need to be considered in depth. Here are two aspects which Dr. Hucka talked to me. I think it is really worth digging:
```

Comment:
Probably should not name specific potential mentors in the proposals. Rather, say something along the lines of "based on discussions with the project organizers", etc.